pkg/minikube/registry/drvs/virtualbox/virtualbox.go

```diff
@@ -52,6 +52,11 @@ func init() {
 }
 
 func configure(cc config.ClusterConfig, n config.Node) (interface{}, error) {
+	hostOnlyCIDR = strings.Trim(hostOnlyCIDR, "'"")
```

Comment:
@Aut0R3V looks like you forgot to remote the double quote?

Reply:
The build reports:
```
pkg/minikube/registry/drvs/virtualbox/virtualbox.go:55:47: syntax error: unexpected literal "), expecting comma or )
pkg/minikube/registry/drvs/virtualbox/virtualbox.go:55:49: newline in string
```

Reply:
Sure, I'll update. Thanks